release/models/aft/openconfig-aft-common.yang

```diff
@@ -22,7 +22,13 @@ submodule openconfig-aft-common {
     "Submodule containing definitions of groupings that are re-used
     across multiple contexts within the AFT model.";
 
-  oc-ext:openconfig-version "0.8.0";
+  oc-ext:openconfig-version "0.9.0";
+
+  revision "2021-10-20" {
+    description
+      "Add atomic attribute to the next-hops container.";
```

Comment:
This revision description should be common across all the files, rather than one per file.